script/replication/replication_procedure_task.sql

```diff
@@ -30,14 +36,6 @@ CREATE OR REPLACE TASK replicate_history_query
 AS
 CALL REPLICATE_HISTORY_QUERY('UNRAVEL_SHARE','SCHEMA_4823_T',2);
 
-
--- create warehouse replicate Task
```

Comment:
Can you keep this task as well? If something goes wrong with RT_Query task, then warehouse data not available for Query_History also. @viswa-unravel pls help to confirm this behavior.